cmd/osv-scanner/scan/source/__snapshots__/command_test.snap

```diff
@@ -1467,6 +1467,7 @@ Total 0 packages affected by 0 known vulnerabilities (0 Critical, 0 High, 0 Medi
 
 [TestCommand_ExplicitExtractors/scanning_directory_with_an_extractor_that_does_not_exist - 2]
 unknown extractor "custom/extractor"
+unknown extractor "custom/extractor"
```

Comment:
This is being printed twice now as it's printed by `builders.BuildExtractors` which gets called when we resolve the enabled/disabled extractors which we do for the lockfiles section and then again for the directories section, whereas previously we'd handle that once in the command.

While not ideal I don't think it's worth blocking this since it should only happen for people trying to enable specific extractors and its an error anyway so they'll have to change their input no matter how many times we print this message.

This will eventually get addressed through some of the follow-up changes, probably even before the next release anyway 🤷

Reply:
Actually, this should be taken care of by #2102 which can be landed before or after this one